gradio/static/js/gradio.js

```diff
@@ -51,10 +51,14 @@ function gradio(config, fn, target, example_file_path) {
       <button class="run_examples">Run All</button>
       <button class="load_prev">Load Previous <em>(CTRL + &larr;)</em></button>
       <button class="load_next">Load Next <em>(CTRL + &rarr;)</em></button>
-      <button class="order_similar">Order by Similarity</em></button>
+      <button class="order_similar">Order by Similarity</button>
+      <button class="view_embeddings">View Embeddings</button>
```

Comment:
will these embedding buttons and functionality be visible even if embeddings hasn't been enabled?

Reply:
Yes because embeddings is enabled by default when examples are